{{cookiecutter.project_slug}}/ansible/provision.yml

```diff
@@ -7,59 +7,66 @@
   become: yes
   tasks:
 
-    - name: Add user "buildout" and set an unusable password
-      user: name=buildout password='*' state=present shell="/bin/bash"
+    - name: Add user "buildout" and set an unusable password.
+      user:
+        name: buildout
+        password: '*'
+        state: present
+        shell: "/bin/bash"
 
     - name: "Add maintainers' ssh keys so they can log in as user buildout."
-      authorized_key: user=buildout key=https://github.com/{% raw %}{{ item}}{% endraw %}.keys
+      authorized_key:
+        user: buildout
+        key: https://github.com/{{ item }}.keys
       with_items:
-        - reinout
-        - TODO
+        - !!TODO.fill.in.your.username
```

Comment:
`byrman` hier bijvoorbeeld alvast bijzetten? Remco. Dat is wel makkelijk voor de "oh oepsie" releases als er iemand anders met vakantie is.